15.04.2024 17-09-30 ERROR C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\utils\log.py Internal Server Error: /post/search/ (<class 'django.template.exceptions.TemplateDoesNotExist'>, TemplateDoesNotExist('news/post_list.html'), <traceback object at 0x000001D71DAAAE80>)
Traceback (most recent call last):
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: news/post_list.html
15.04.2024 17-09-30 ERROR C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\servers\basehttp.py "GET /post/search/ HTTP/1.1" 500 86903 None
15.04.2024 17-09-54 ERROR C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\utils\log.py Internal Server Error: /post/search/ (<class 'django.template.exceptions.TemplateDoesNotExist'>, TemplateDoesNotExist('news/post_list.html'), <traceback object at 0x000001D71D65C380>)
Traceback (most recent call last):
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 90, in rendered_content
    template = self.resolve_template(self.template_name)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\response.py", line 72, in resolve_template
    return select_template(template, using=self.using)
  File "C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\template\loader.py", line 47, in select_template
    raise TemplateDoesNotExist(", ".join(template_name_list), chain=chain)
django.template.exceptions.TemplateDoesNotExist: news/post_list.html
15.04.2024 17-09-54 ERROR C:\Users\alext\Desktop\Python\venvs\News\lib\site-packages\django\core\servers\basehttp.py "GET /post/search/ HTTP/1.1" 500 87040 None
